doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -15,108 +15,24 @@ For end to end examples leveraging RaySGD TorchTrainer, jump to :ref:`raysgd-tor
 
 .. contents:: :local:
 
+Basic Usage
```

Comment:
@richardliaw any recommendations for a better section header here?

Reply:
Looks fine for now